introduction.md

```diff
@@ -0,0 +1 @@
+introducing the second attempt to make this codeacademy course work
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.